test/fixtures/poro.rb

```diff
@@ -258,4 +260,34 @@ def maker
   cache only: [:id]
   attributes :id
 end
+
+class TweetSerializer < ActiveModel::Serializer
+  attributes :id, :body, :date
+
+  belongs_to :author
+  class AuthorSerializer < ActiveModel::Serializer
+    attributes :id
+  end
```

Comment:
It is, indeed syntactic sugar. However, I could see in the future having, say the Json adapter default to including all nested relationships (default to `**` instead of `*` currently), in a kind of safe way since the user would provide all the subsequent serializers to make the graph explicit (maybe with a warning when there is an undefined serializer and possibly when using a fallback serializer).